.cirrus.yml

```diff
@@ -26,14 +26,14 @@ env:
     ####
     #### Cache-image names to test with (double-quotes around names are critical)
     ####
-    FEDORA_NAME: "fedora-38"
+    FEDORA_NAME: "fedora-39"
```

Comment:
WDYT is it worth classifying this as `fedora-39 (beta)` or some-such?

I believe it will still [pass the distro-verison check](https://github.com/containers/podman/pull/20162/files#diff-f3866c83e5fdf3650df2ad5b9c0c18efaa71891d23675732cc931e90fd2b10faR66-R69) and should display in task-names as a reminder.

I'm not going to block over this, since it could be done as a follow-up.  I'm feel it may be an important visual reminder that somebody needs to do another F39 build whenever it's released.

Reply:
Ooh, I like that. I'd prefer to do it as a followup, because I don't know how/if it'd affect logformatter URLs.

Reply:
Seems to work! In [urls](https://api.cirrus-ci.com/v1/artifact/task/5308206646296576/html/sys-remote-fedora-39%CE%B2-root-host-sqlite.log.html) and in github display: 
![beta](https://github.com/containers/podman/assets/265639/d74475af-278f-43c0-9e89-b8a518244164)

Re-pushed with β